src/Tools/GenerateProps/targetgroups.props

```diff
@@ -116,11 +116,16 @@
       <NuGetTargetMoniker>WindowsPhoneApp,Version=v8.1</NuGetTargetMoniker>
     </TargetGroups>
     <TargetGroups Include="uap101aot">
+      <TargetingPackNugetPackageId>Microsoft.TargetingPack.Private.NetNative</TargetingPackNugetPackageId>
```

Comment:
I'm not sure, for now I just copied it from the netcore50 targetgroup section, since I thought it might come into play when building the actual src\src.builds. If it is not necessary then I'll remove it.